src/main/webapp/WEB-INF/static/js/trialmanager/manageTrial.js

```diff
@@ -842,6 +707,11 @@ stockListImportNotSaved, ImportDesign, isOpenStudy, displayAdvanceList, Inventor
 							}
 						})
 					});
+					// Add crosses and selections tab after Observations tab.
+					// datasetType id 4 is plot type.
+					if (datasetType.id === 4) {
+						$scope.subObservationTabs.push($scope.crossesAndSelectionsTab);
+					}
```

Comment:
Perhaps we can add them all after the obs-type tabs (obs and sub-obs), and keep the list in the template
```html
<li ng-repeat="tab in crossesAndSelectionsTab
```
The mockup show them after observation tab but it doesn't mention sub-obs...
Pushing them to the list of obs tabs feels wrong to me.